scripts/examples/receive.clje

```diff
@@ -38,4 +38,4 @@
   (erlang/send x :dec)
   (erlang/send x :print)
   (erlang/send x #erl[:get self])
-  (receive* x (prn x)))
+  (receive* msg (prn msg)))
```

Comment:
This was changed because there is an existing binding for `x` which made the `receive*` block forever because the message didn't match.